Source/WebKit/UIProcess/RemoteLayerTree/mac/RemoteLayerTreeEventDispatcher.h

```diff
@@ -100,12 +101,15 @@ class RemoteLayerTreeEventDispatcher : public ThreadSafeRefCounted<RemoteLayerTr
     void wheelEventWasHandledByScrollingThread(WheelEventHandlingResult);
 
     DisplayLink* displayLink() const;
+    DisplayLink* exisingDisplayLink() const;
```

Comment:
typo: existing

Reply:
oh, the name is actually copied from the existent `emoteLayerTreeDrawingAreaProxyMac::exisingDisplayLink()`. So I'll also fix these ones if you don't mind.